assets/dev/js/frontend/utils/anchors.js

```diff
@@ -7,7 +7,7 @@ module.exports = elementorModules.ViewModule.extend( {
 			selectors: {
 				links: 'a[href*="#"]',
 				targets: '.elementor-element, .elementor-menu-anchor',
-				scrollable: 'html, body',
+				scrollable: isScrollSnapActive() ? 'body' : 'html, body',
```

Comment:
If there are two elements, it causes the scrolling animation to run twice and it sometimes breaks scroll-snap. In line 75

Reply:
Is the `html` still needed for when there is no scroll snap?

Reply:
Yes, it works only with HTML, I thought maybe to delete the 'body' if scroll-snap is off, but I'm not sure if the 'html, body' together is fixing some sort of bug.